src/components/proximities/proximities.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-unused-vars */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Heading2 } from '../typography/headline/h2';
+import { colors } from '../../typography/colors';
+import { BodySecondary } from '../typography/body/body-secondary';
+import './proximities.css';
+import { Separator } from '../separator/separator';
+import { Divider } from '../cards/divider/divider';
+import {
+  IconDefinition,
+  faCartShopping,
+  faGraduationCap,
+  faHospital,
+  faLeaf,
+  faUtensils,
+} from '@fortawesome/free-solid-svg-icons';
+
+const proximitiesData = {
+  proximities: [
+    {
+      type: 'WellRatedRestaurants',
```

Comment:
poderia usar os enums aqui ne

```suggestion
      type: ProximitiesType.WellRatedRestaurants',
```